packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -134,6 +158,12 @@ export class Tooltip<
                     },
                 }}
                 {...restProps}
+                // eslint-disable-next-line react/jsx-no-bind
```

Comment:
nit: preference for `useCallback` over disabling the lint rule

Reply:
done